sdk-extension/opentelemetry-sdk-extension-aws/tests/trace/propagation/test_aws_xray_format.py

```diff
@@ -372,3 +373,35 @@ def test_extract_invalid_non_numeric_sampled_flag(self):
             get_nested_span_context(context_with_extracted),
             INVALID_SPAN_CONTEXT,
         )
+
+    @patch(
+        "opentelemetry.sdk.extension.aws.trace."
+        "propagation.aws_xray_format.trace"
+    )
+    def test_fields(self, mock_trace):
```

Comment:
@ocelotl Thanks for adding this test! Although I thought I already added a test for this in #228? Should we just keep 1 test?

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/e5a0153794e30694248e9d8f24138e27461337e5/sdk-extension/opentelemetry-sdk-extension-aws/tests/trace/propagation/test_aws_xray_format.py#L157-L170

Reply:
Since these 2 tests use a different setup method, I would leave them both, they may end up finding a bug just because of this difference.